lib/src/view/game/archived_game_screen.dart

```diff
@@ -44,7 +45,12 @@ class ArchivedGameScreen extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     if (gameData != null) {
-      return _Body(gameData: gameData, orientation: orientation, initialCursor: initialCursor);
+      return _Body(
+        gameData: gameData,
+        orientation: orientation,
+        isLoggedIn: ref.watch(isLoggedInProvider),
```

Comment:
Why adding a new parameter here? Seems that the `isLoggedIn` is only used in the `_Body()` so you can watch it directly here.